erpnext/accounts/doctype/pos_invoice_merge_log/pos_invoice_merge_log.py

```diff
@@ -241,6 +245,10 @@ def merge_pos_invoice_into(self, invoice, data):
 		invoice.disable_rounded_total = cint(
 			frappe.db.get_value("POS Profile", invoice.pos_profile, "disable_rounded_total")
 		)
+		accounting_dimensions = get_accounting_dimensions()
+		for dimension in accounting_dimensions:
+			dimension_value = frappe.db.get_value("POS Profile", invoice.pos_profile, dimension)
```

Comment:
You can get all values in one call, no need to do multiple queries

Reply:
fixed it